go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/cilium/certgen
 
-go 1.15
+go 1.16
```

Comment:
Is there a reason to bump the Go version in the `go.mod`?

Reply:
Why would we want not to do it?

Reply:
I don't see a reason to bump it unless we use Go 1.16 features. Certgen is relatively self-contained but in general, and especially for libraries, I would refrain from bumping the version unless there's a reason to (e.g. making use of Go 1.16 features).

Reply:
But then since we don't actually test with Go v1.15 it's easy to break right (by using a Go v1.16 feature)?

Reply:
In the go.mod, this statement is mostly about changing the behavior of the `go` tool: https://golang.org/ref/mod#go-mod-file-go

While it's not necessary, I think it's fine to bump it. We're not building certgen out of tree to my knowledge, so we control pretty much the build via `Dockerfile`.